### DISCUSSION TITLE
使用空调网关(VRF)时的室内机温度无法在苹果家庭 app 中直观显示

### DISCUSSION THREAD

**Discussion by laguan799** [XiaoMi/ha_xiaomi_home#279] [General]
原使用 Miot 时，苹果 app 的房间空调控制标签中显示的温度为室内温度。使用官方插件后，该温度是个固定值，无法改变。想看室内温度需进入温度子项查看。
![Uploading 截屏2024-12-20 13.52.14.png…]()

**Comment by laguan799** [comment]
![截屏2024-12-20 13 52 14](https://github.com/user-attachments/assets/36019e68-e2a6-4f03-b86c-557a0e19b9a6)

**Comment by SusanPhevos** [comment]
您好！请发一下设备详情页，我们看看是不是设备的问题。

**Reply by JimmyKmi** [reply]
似乎还是因为空调实体不能绑定温湿度传感器的原因，这个已经在排期了

**Reply by JimmyKmi** [reply]
同
https://github.com/XiaoMi/ha_xiaomi_home/issues/137
https://github.com/XiaoMi/ha_xiaomi_home/discussions/337
